Doc/c-api/long.rst

```diff
@@ -43,8 +43,7 @@ distinguished from a number.  Use :c:func:`PyErr_Occurred` to disambiguate.
    The current implementation keeps an array of integer objects for all integers
    between ``-5`` and ``256``, when you create an int in that range you actually
    just get back a reference to the existing object. So it should be possible to
-   change the value of ``1``.  I suspect the behaviour of Python in this case is
-   undefined. :-)
+   change the value of ``1``.
```

Comment:
Meant to include the removal of this line in my previous review, part of the previous sentence I was referring to:
```suggestion
```